agent/src/Handler/Apps.hs

```diff
@@ -303,6 +303,8 @@ getInstalledAppByIdLogic appId = do
                 , appInstalledFullLastBackup             = backupTime
                 , appInstalledFullTorAddress             = Nothing
                 , appInstalledFullLanAddress             = Nothing
+                , appInstalledFullTorUi                  = False
+                , appInstalledFullLanUi                  = False
```

Comment:
These needed to be added to the full response. Not sure why they weren't there before, and also not sure why it worked, but it did.